pages/landing.tsx

```diff
@@ -0,0 +1,277 @@
+  const { t } = useTranslation('common');
+
+  return (
+    <Layout>
+      <GridItem colSpan={8} colStart={1} mt="64px">
+        <Text as="h1" textStyle="display-xlarge" align="left">
+          {t('landing')}
+        </Text>
+        <Text as="p" textStyle="display-small" align="left" mt="24px">
+          Find the services you need related to your British Columbia accessible parking permit. You
+          may chose to use our online service or download a physical form and email, mail or drop it
+          off in person to RCD.{' '}
+        </Text>
+      </GridItem>
+      <GridItem colSpan={5} colStart={1} mt="28px">
+        <Text as="h2" textStyle="display-medium" align="left" fontWeight="semibold">
+          Option 1: Online Service
+        </Text>
+        <Text as="p" textStyle="body-regular" align="left" mt="24px">
+          You will need to complete the following steps for our online services
+        </Text>
+        <UnorderedList mt="12px" pl="10px">
+          <ListItem textAlign="left" textStyle="body-regular">
+            <Text as="p">Provide your user ID (available on your wallet card)</Text>
+          </ListItem>
+          <ListItem textAlign="left" textStyle="body-regular">
+            <Text as="p">Provide the last 4 digits of your phone number</Text>
+          </ListItem>
+          <ListItem textAlign="left" textStyle="body-regular">
+            <Text as="p">Provide your date of birth</Text>
+          </ListItem>
+          <ListItem textAlign="left" textStyle="body-regular">
+            <Text as="p">Complete the online form and pay a $26 processing fee</Text>
+          </ListItem>
+        </UnorderedList>
+      </GridItem>
+      <GridItem colSpan={5} colStart={7} mt="28px">
+        <Text as="h2" textStyle="display-medium" align="left" fontWeight="semibold">
+          Option 2: Download Form
+        </Text>
+        <Text as="p" textStyle="body-regular" align="left" mt="24px">
+          If you would prefer to fill out a replacement form for a lost or stolen parking permit by
+          hand, or want to request a new parking permit, please download the appropriate form below.
+          <br></br>
+          <br></br>
+          After completing the form, either email (
+          <b>
+            <a href="mailto:parkingpermit@richmond.org">parkingpermit@rcdrichmond.org</a>
+          </b>
+          ), mail or drop it off in person to RCD!
+        </Text>
+      </GridItem>
+
+      <GridItem colSpan={5} colStart={1} textAlign="left" mt="12px">
+        <Link href="#" style={{ textDecoration: 'none' }}>
+          <Button
+            colorScheme="primary"
+            variant="solid"
+            fontWeight="normal"
+            size="lg"
+            width="320px"
+            height="48px"
+            radius="6px"
+            p="12px 25px 12px 25px"
+          >
+            <Text as="span">Renew your permit online</Text>
+          </Button>
+        </Link>
+      </GridItem>
```

Comment:
The content in this `GridItem` should be combined with the content in the `GridItem` with the description of Option 1. Same with the `GridItem` after this one and that with the description of Option 2.